spatial-history/index.md

```diff
@@ -1,39 +1,120 @@
+</p>
+
+<p>
+For a company to succeed in the early stages, it is important to be financially stable. Companies require to maximize their profit to continue to invest in their own company. Toronto is a cheaper alternative to Silicon Valley. Toronto provides cheaper rent for companies and their employee. Toronto is currently 58.27% cheaper than in Silicon Valley [<a href="https://www.numbeo.com/cost-of-living/compare_cities.jsp?country1=United+States&country2=Canada&city1=San+Francisco%2C+CA&city2=Toronto&tracking=getDispatchComparison">7</a>]. To rent office space and their home can accumulate quickly. It is also cheaper to hire interns/employees in Toronto than in Silicon Valley. A founder of a startup researched the cost difference between Toronto and Silicon Valley and found that there is approximately 4 times the difference in cost.
+<blockquote>
+<em>
+“Comparing these numbers — $190k USD for a developer in San Francisco to $48k USD for a developer in Toronto — we are looking at a ~4x difference. In other words, you could hire 4 developers in Toronto for the price of 1 in SF!” [<a href="https://medium.com/@hussein_fazal/where-to-build-your-next-tech-startup-toronto-vs-san-francisco-bff95b2ccb91">8</a>]
+</em>
+</blockquote>
+Hiring 4 developers for the cost of 1 can result in faster development and faster results for the company. For a startup, speed is essential since there are so many startups attempting to race their product to market. Not only is it smart for employers to choose Toronto for cost, however for employees too. An article from Planet Web has reported that Toronto’s salaries are 11% more than San Francisco when adjusted for cost of living [<a href="http://www.planetweb.ca/news/toronto-salaries-11-san-francisco-adjusted-cost-living-hired-reports/ ">9</a>]. Even though San Francisco’s salaries are higher, the cost of living is also 58% higher. Toronto is clearly the strategic spot for a startup when the cost of operation is in consider.
+</p>
 
-The rest of your essay should go under the map, probably.
+<p>
+Toronto demonstrates all the key factors that Silicon Valley had for their success. Producing an abundance of skilled workers with their infrastructure with the help of Toronto’s unique city. The city provides the correct support a startup requires and emerges the companies with a diverse culture. The cost of operation being cheaper than Silicon Valley can assist the companies to push their product to market first before their competition. Toronto provides the correct environment for any tech companies to thrive and become a billion dollar company.
+</p>
 
-## in sections like this
-also _italics_ and __bold__ which can also be written *like this* or **like this**
-etc. Be sure to [make use of links](http://digital.hackinghistory.ca) -- that's one of the reasons we write on the Web, and I've asked you to use links for all yor footnotes/references as well.  
+Sources:
```

Comment:
This essay is pretty good but needs to be rewritten relying on scholarly sources. There as a very substantial literature on the geographical factors associated with the rise of Silicon Valley. cf [this simple search](https://scholar-google-ca.myaccess.library.utoronto.ca/scholar?hl=en&as_sdt=0%2C5&q=geography+silicon+valley+toronto&btnG=). So, basically, I want you to do the historical/scholarly research and plug new quotes into the appropriate places. I will be working you pretty hard on these elements of hte assignments from now on!